packages/kununu-theme-v2/scss/base/type.scss

```diff
@@ -89,22 +89,12 @@ p {
   color: $gray-base-30;
 }
 
-// Still used in insights
-.text-semibold {
-  font-weight: $font-weight-semibold;
-}
-
-// Still used in insights
-.text-base {
-  font-size: $font-size-base;
-}
-
 // Still used in mykununu
 .text-sm {
   font-size: $font-size-sm;
 }
 
-// Still used in insights
+// Still used in mykununu and profiles specs
```

Comment:
Still used in footer, that's why is only in specs

Reply:
@clopes-ventureoak Keep it here them?

Reply:
Change the comment...